akka-remote/src/main/resources/reference.conf

```diff
@@ -49,6 +49,7 @@ akka {
       "com.google.protobuf.GeneratedMessage" = proto
       
       "java.util.Optional" = akka-misc
+      "akka.Done" = akka-misc
```

Comment:
We can't enable this by default in 2.5.x because it would break rolling upgrades. That must be done in 2.6.0 with migration guide for rolling upgrade. It can be done in Lagom play overrides if Lagom is less strict about that.

The serializer can be added but this config should be removed and instead be documented. In 2.4 we had a convenience property to enable a bunch of serializers, `enable-additional-serialization-bindings = true`. We could do the same in 2.5, `enable-additional-serialization-bindings2`, but since we don't know if there will be any more than this `Done` we could start without that and just document that `akka.Done` can be enabled.

cc @TimMoore

Reply:
What we've done elsewhere in Lagom is back-port the serializer to the stable branch, but disabled, so that the latest stable is capable of deserializing the new format but does not use it to serialize by default.

Then, the next version uses it by default. That way, you can do a rolling upgrade from the latest patch of the previous stable version (e.g., 1.3.10) to the new release (1.4.0).

See https://www.lagomframework.com/documentation/1.4.x/java/Migration14.html# Rolling-upgrade for an example (`ClusterStartupTaskActor$Execute`).

What will make this tricky to enable by default for Lagom 1.4 is that we would need to backport this to Akka 2.4 in order to make it available in Lagom 1.3.x.

We may need to leave this opt in for the time being and document how to enable it.

Reply:
@patriknw This makes sense. I can make the change in a few hours. Does it make sense to eventually make this serialization default?

Reply:
yes, it can be enabled by default in 2.6.0, but not in 2.5.x

Reply:
https://github.com/akka/akka/issues/20507 contains a list of additional serializers that it would be good to have for, perhaps enough to justify a new config. (`NotUsed`, `None`, `Some` - though I thought `Option` was already handled?).

Reply:
@erip Looks like this was not changed? It should be removed here and instead added to a new section in docs https://github.com/akka/akka/blob/master/akka-docs/src/main/paradox/serialization.md

It could be mentioned in reference.conf also as a comment

Reply:
ping @erip ?

Reply:
Sorry -- been super busy with work and grad school as the term winds down. I'll address this ASAP. 👍